app/components/join-section.hbs

```diff
@@ -41,13 +41,12 @@
     {{!-- NOTE: This has the new code on feature flag off --}}
     {{!-- If something goes wrong, we can enable a flag to restore previous behaviour --}}
     {{!-- TODO: Clean up by Jan, 2025 --}}
+    {{#if this.isDevMode}}
     <p data-test-join-later-text class='join__later-text'>
-      {{#if this.isDevMode}}
       We're currently not accepting new applications. Please come back later.
```

Comment:
Since the join section is styled separately from [subscribe page](https://github.com/Real-Dev-Squad/website-www/pull/994/files#diff-05efdc7e5350b3b112828cb661de492625904672c634d85e702321f8f4c63a38R43), have to pass the individual classes to the common notice component for applying the specific styles